lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1792,9 +1794,10 @@ def _resource_sku_to_capability(  # noqa: C901
             # Check if CachedDiskBytes is greater than 30GB
             # We use diff disk as cache disk for ephemeral OS disk
             cached_disk_bytes = azure_raw_capabilities.get("CachedDiskBytes", 0)
-            cached_disk_bytes_gb = int(cached_disk_bytes) / 1024 / 1024 / 1024
+            cached_disk_bytes_gb = int(int(cached_disk_bytes) / 1024 / 1024 / 1024)
```

Comment:
```suggestion
            cached_disk_bytes_gb = int(cached_disk_bytes / 1024 / 1024 / 1024)
```

Reply:
cached_disk_bytes  is a str. the change like this has an error.